core-plugins/src/test/java/io/cdap/plugin/batch/ETLTPFSTestRun.java

```diff
@@ -271,11 +250,6 @@ public void testAvroSourceConversionToAvroSink() throws Exception {
       .setOutputFormat(AvroKeyOutputFormat.class)
       .setInputProperty("schema", avroSchema.toString())
       .setOutputProperty("schema", avroSchema.toString())
-      .setEnableExploreOnCreate(true)
-      .setSerDe("org.apache.hadoop.hive.serde2.avro.AvroSerDe")
-      .setExploreInputFormat("org.apache.hadoop.hive.ql.io.avro.AvroContainerInputFormat")
-      .setExploreOutputFormat("org.apache.hadoop.hive.ql.io.avro.AvroContainerOutputFormat")
-      .setTableProperty("avro.schema.literal", (avroSchema.toString()))
```

Comment:
Do we need to remove this?

Reply:
Yes.`FileSetProperties.java` doesn't contain these methods anymore as they are deleted from the source file in cdap repository.

Reply:
Added it back